### ISSUE TITLE
cgroup-network-helper.sh issues in our Docker images

### ISSUE THREAD

**Issue by cakrit** [netdata/netdata#5880]
We seem to have some errors with our cgroup-network scripts when they are running from inside our docker images. 

The following for example complains at least about not finding `lsns`:
```
[christopher@chris-msi ~]$ docker run -d --name=netdatatest -p 19998:19999 -v /proc:/host/proc:ro -v /sys:/host/sys:ro -v /var/run/docker.sock:/var/run/docker.sock:ro --cap-add SYS_PTRACE --security-opt apparmor=unconfined -e PGID=1005 netdata/netdata netdata -s /host
...
[christopher@chris-msi ~]$ sudo docker logs netdatatest 2>&1 | grep network
2019-04-17 07:35:24: /usr/libexec/netdata/plugins.d/cgroup-network INFO  : MAIN : Using host prefix directory '/host'
2019-04-17 07:35:24: /usr/libexec/netdata/plugins.d/cgroup-network INFO  : MAIN : running: exec /usr/libexec/netdata/plugins.d/cgroup-network-helper.sh --cgroup '/host/sys/fs/cgroup/cpu,cpuacct/machine.slice/machine-qemu\x2d1\x2dfedora29.scope'
2019-04-17 07:35:25: cgroup-network-helper.sh: INFO: searching for network interfaces of cgroup '/host/sys/fs/cgroup/cpu,cpuacct/machine.slice/machine-qemu\x2d1\x2dfedora29.scope'
2019-04-17 07:35:25: /usr/libexec/netdata/plugins.d/cgroup-network ERROR : MAIN : child pid 340 exited with code 1.
2019-04-17 07:35:25: /usr/libexec/netdata/plugins.d/cgroup-network ERROR : MAIN : Cannot find a cgroup PID from cgroup '/host/sys/fs/cgroup/cpu,cpuacct/machine.slice/machine-qemu\x2d1\x2dfedora29.scope'
2019-04-17 07:35:34: /usr/libexec/netdata/plugins.d/cgroup-network INFO  : MAIN : Using host prefix directory '/host'
2019-04-17 07:35:34: /usr/libexec/netdata/plugins.d/cgroup-network INFO  : MAIN : running: exec /usr/libexec/netdata/plugins.d/cgroup-network-helper.sh --cgroup '/host/sys/fs/cgroup/cpu,cpuacct/docker/29102fd49bc16168fdfc2f66ed0f7c312942d72a2ed3197822c850444347ea8f'
2019-04-17 07:35:35: cgroup-network-helper.sh: INFO: searching for network interfaces of cgroup '/host/sys/fs/cgroup/cpu,cpuacct/docker/29102fd49bc16168fdfc2f66ed0f7c312942d72a2ed3197822c850444347ea8f'
/usr/libexec/netdata/plugins.d/cgroup-network-helper.sh: line 196: lsns: command not found
/usr/libexec/netdata/plugins.d/cgroup-network-helper.sh: line 196: lsns: command not found
/usr/libexec/netdata/plugins.d/cgroup-network-helper.sh: line 196: lsns: command not found
/usr/libexec/netdata/plugins.d/cgroup-network-helper.sh: line 196: lsns: command not found
/usr/libexec/netdata/plugins.d/cgroup-network-helper.sh: line 196: lsns: command not found
/usr/libexec/netdata/plugins.d/cgroup-network-helper.sh: line 196: lsns: command not found
/usr/libexec/netdata/plugins.d/cgroup-network-helper.sh: line 196: lsns: command not found
/usr/libexec/netdata/plugins.d/cgroup-network-helper.sh: line 196: lsns: command not found
2019-04-17 07:35:35: /usr/libexec/netdata/plugins.d/cgroup-network ERROR : MAIN : child pid 373 exited with code 1.
2019-04-17 07:35:35: /usr/libexec/netdata/plugins.d/cgroup-network ERROR : MAIN : Cannot switch to network namespace of pid 1 (errno 1, Operation not permitted)
2019-04-17 07:35:35: /usr/libexec/netdata/plugins.d/cgroup-network ERROR : MAIN : Cannot switch to pid namespace of pid 1 (errno 1, Operation not permitted)
2019-04-17 07:35:35: /usr/libexec/netdata/plugins.d/cgroup-network ERROR : MAIN : Cannot switch to mount namespace of pid 1 (errno 1, Operation not permitted)
2019-04-17 07:35:35: /usr/libexec/netdata/plugins.d/cgroup-network ERROR : MAIN : Cannot read '/sys/class/net/eth0/ifindex'. (errno 2, No such file or directory)
2019-04-17 07:35:35: /usr/libexec/netdata/plugins.d/cgroup-network ERROR : MAIN : Cannot read '/sys/class/net/eth0/iflink'. (errno 2, No such file or directory)
2019-04-17 07:35:35: /usr/libexec/netdata/plugins.d/cgroup-network ERROR : MAIN : there are not double-linked cgroup interfaces available.
2019-04-17 07:35:35: /usr/libexec/netdata/plugins.d/cgroup-network INFO  : MAIN : Using host prefix directory '/host'
2019-04-17 07:35:35: /usr/libexec/netdata/plugins.d/cgroup-network INFO  : MAIN : running: exec /usr/libexec/netdata/plugins.d/cgroup-network-helper.sh --cgroup '/host/sys/fs/cgroup/cpu,cpuacct/docker/6ec4274250abf86f0710ca3528330a44c3e946fec23950775dcf037591b198fa'
2019-04-17 07:35:36: cgroup-network-helper.sh: INFO: searching for network interfaces of cgroup '/host/sys/fs/cgroup/cpu,cpuacct/docker/6ec4274250abf86f0710ca3528330a44c3e946fec23950775dcf037591b198fa'
2019-04-17 07:35:36: /usr/libexec/netdata/plugins.d/cgroup-network ERROR : MAIN : child pid 419 exited with code 1.
2019-04-17 07:35:36: /usr/libexec/netdata/plugins.d/cgroup-network ERROR : MAIN : Cannot find a cgroup PID from cgroup '/host/sys/fs/cgroup/cpu,cpuacct/docker/6ec4274250abf86f0710ca3528330a44c3e946fec23950775dcf037591b198fa'
```

From within the container, I get the following:
```
[christopher@chris-msi ~]$ sudo docker exec -it netdatatest bash
bash-4.4# export NETDATA_CGROUP_NETWORK_HELPER_DEBUG=1
bash-4.4# usr/libexec/netdata/plugins.d/cgroup-network-helper.sh --cgroup '/host/sys/fs/cgroup/cpu,cpuacct/docker/6ec4274250abf86f0710ca3528330a44c3e946fec23950775dcf037591b198fa'
2019-04-17 12:38:14: cgroup-network-helper.sh: INFO: searching for network interfaces of cgroup '/host/sys/fs/cgroup/cpu,cpuacct/docker/6ec4274250abf86f0710ca3528330a44c3e946fec23950775dcf037591b198fa'
2019-04-17 12:38:14: cgroup-network-helper.sh: DEBUG: Cannot find file '/host/sys/fs/cgroup/cpu,cpuacct/docker/6ec4274250abf86f0710ca3528330a44c3e946fec23950775dcf037591b198fa/emulator/cgroup.procs', not searching for tun/tap interfaces.
2019-04-17 12:38:14: cgroup-network-helper.sh: DEBUG: virsh command is not available
2019-04-17 12:38:14: cgroup-network-helper.sh: DEBUG: found 0 network interfaces for pid '', cgroup '/host/sys/fs/cgroup/cpu,cpuacct/docker/6ec4274250abf86f0710ca3528330a44c3e946fec23950775dcf037591b198fa', run as , 0
```

It looks like we need `lsns` and/or `virsh` in our image, but I haven't gone deeper with this script.

**Comment by kam1kaze** [comment]
I've created PR to fix issue with `lsns` netdata/helper-images#1

**Comment by cakrit** [comment]
Thanks @kam1kaze!
